src/main/java/com/comphenix/protocol/wrappers/WrappedLevelChunkData.java

```diff
@@ -163,11 +164,17 @@ public static class LightData extends AbstractWrapper {
                     MinecraftReflection.getPacketDataSerializerClass(), int.class, int.class);
             BIT_SET_ACCESSORS = Accessors.getFieldAccessorArray(HANDLE_TYPE, BitSet.class, true);
             BYTE_ARRAY_LIST_ACCESSORS = Accessors.getFieldAccessorArray(HANDLE_TYPE, List.class, true);
-            TRUST_EDGES_ACCESSOR = Accessors.getFieldAccessor(reflection.getField(FuzzyFieldContract.newBuilder()
-                    .typeExact(boolean.class)
-                    .build()));
+            if(MinecraftVersion.TRAILS_AND_TAILS.atOrAbove()) {
+                TRUST_EDGES_ACCESSOR = null;
+            } else {
+                TRUST_EDGES_ACCESSOR = Accessors.getFieldAccessor(reflection.getField(FuzzyFieldContract.newBuilder()
+                        .typeExact(boolean.class)
+                        .build()));
+            }
         }
 
+        private boolean dummyTrustEdges = false;
```

Comment:
Not sure about this one... Do we do this in all wrappers that don't support a particular setter anymore?

Reply:
I was unsure on this as well. The getter and setter should remain I guess. Storing the dummy value ensures that programs using the deprecated functions does not cause any inconsistencies. 
@dmulloy2 What do you think about this?

Reply:
i don't think it's been done before, but it makes sense to me